_gsocorgs/2022/ijclab.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Laboratoire de Physique des 2 infinis Irène Joliot-Curie (IJCLab)"
+author: "Julien Peloton"
```

Comment:
It is strange to see here Julien's name as he is not involved in GSoC this year? Maybe, if a name is needed, Hadrien's name is more appropriate?

Reply:
I added Hadrien now